app/src/main/java/Model/Node.java

```diff
@@ -29,6 +29,15 @@ public Node(String tag) {
      * {@link Model.OutputPort OutputPorts}.
      */
     public abstract void evaluate();
+    
+    /**
+     * Sets the value of all the "OutputPorts" in "outputs"
+     * @author Viktor Bicskei
+     * @param value
+     */
+    public void setOutputs(boolean value) {
+    	outputs.stream().forEach(x -> x.setValue(value));
```

Comment:
Is it possible to have mode that can have different output values of different output nodes?

Reply:
Are you thinking of something along the lines of:
```
public void setOutputs(Iterator<Boolean> values) {
    ...
}
```

Reply:
@bgdxqi yes, that is what I meant

Reply:
> But perhaps I can rename it to SetAllOutputs ...

I think that would be a good idea.